styleguide/components/31-molecules/islands/gentinfo-island/_gentinfo_island.scss

```diff
@@ -63,8 +73,24 @@
   }
 
   .gentinfo-chat-button {
+    display: flex;
```

Comment:
This is a button, shouldn't this follow the button atom styling?
It's the first 'in context' implementation of a button and we are already not using the atom styling...

Reply:
Changed the button with icon implementation.